enterprise/frontend/src/metabase-enterprise/database_routing/DestinationDatabasesList/DestinationDatabasesList.tsx

```diff
@@ -61,7 +61,8 @@ export const DestinationDatabasesList = ({
                 td="underline"
                 to={Urls.viewDestinationDatabases(primaryDatabaseId)}
               >
-                View all {destinationDatabases.length}
+                {c("{0} is a number")
```

Comment:
nit: I'd mention that the number describes databases since that may affect how the word 'all' is translated. For example, I think in Spanish it might need to be 'las' rather than 'los' because what's being counted is _databases_ (a feminine noun).

Reply:
How about:

`{0} is a number. This text is short for "View all N databases."`